src/names/common.h

```diff
@@ -5,10 +5,12 @@
 #ifndef H_BITCOIN_NAMES_COMMON
 #define H_BITCOIN_NAMES_COMMON
 
+#include <base58.h>
 #include <compat/endian.h>
 #include <primitives/transaction.h>
 #include <script/script.h>
 #include <serialize.h>
+#include <uint256.h>
```

Comment:
Why are these two new `#include`'s needed?  I don't see them used in the `CQueuedTransaction` you added.